backend/app/contact/routes.py

```diff
@@ -0,0 +1,70 @@
+from flask import jsonify, request, abort
+import uuid
+from app import db
+from app.models import Contact
+from . import contact
+
+
+@contact.route("", methods=["POST"])
+def create_contact():
+    #
+    data = request.get_json(force=True)
+    Name = data.get("name")
+    Email = data.get("email")
+    Secondary_email = data.get("secondary_email")
+    Cellphone = data.get("cellphone")
+    Role = data.get("role")
+    Organization = data.get("organization")
+    Neighbourhood = data.get("neighbourhood")
+    Projects = data.get("projects")
+    Contact_type = data.get("contact_type")
+
+    if (
+        # all non-nullable attributes and relationships
+        Name == ""
+        or Email == ""
+        or Cellphone == ""
+        or Organization == ""
+        or Projects == None
+        or Contact_type == None
+    ):
+        abort(400, "Indicated fields cannot be empty")
+
+    new_contact = Contact(
+        name=Name,
+        email=Email,
+        secondary_email=Secondary_email,
+        cellphone=Cellphone,
+        role=Role,
+        organization=Organization,
+        neighbourhood=Neighbourhood,
+        projects=Projects,
+        contact_type=Contact_type,
+    )
+
+    db.session.add(new_contact)
+
+
+@contact.route("/<uuid:id>/edit", methods=["PUT"])
+def edit_contact(id):
+    contact = Contact.query.filter_by(id=id).first()
+    if contact is None:
+        abort(404, "No contact found with specified id")
+
+    db.session.delete(contact)
+    db.session.commit()
+
+    return jsonify(contact.serialize)
+
+
+# Get all projects for a specified id (Issue #15)
+@contact.route("/<uuid:id>/", methods=["GET"])
+def get_all_projects(id):
+    contact = Contact.query.filter_by(id=id).first()
+    if contact is None:
+        abort(404, "No contact found with specified id")
+
+    # might just be overthinking this but this is the correct way to retrieve
+    # the data correct
+    data = contact.serialize
+    return data["projects"]

```

Comment:
@dgiltz If unsure you can write unit tests in the same pr (fill in the summary/test template that celine created) to test whether you're correct or not. I think you're on the right track but unfortunately the `Contact` model's `projects` attribute has not been added into its serialization.